### DISCUSSION TITLE
Function Export to PDF

### DISCUSSION THREAD

**Discussion by starflighter** [senshu/Sozi#692] [Q&A] [answered]
Hi Community,

is there in Sozi a build-in function to export the presentation as pdf?
I was not able to find it. Maybe I need a hint :-).

Thank you in advance

**Comment by ejvindh** [comment, marked as answer]
Hit the "Export"-button (no. 3 from the right), choose Documenttype-PDF, Page-Size, orientation, click Export at the right.
